### ISSUE TITLE
[feature request] support server dart application

### ISSUE THREAD

**Issue by iapicca** [Tienisto/rhttp#19]
### DESCRIPTION

it would be great to be able to use this package in backend,
but at the time of writing [rHTTP](https://pub.dev/packages/rhttp) depends on flutter
so it can't be used

as far as I know [flutter_rust_bridge](https://pub.dev/packages/flutter_rust_bridge) is "flutter" only by name
and can be used in pure dart implementation

### PROPOSAL

support the creation of dart server based on [rHTTP](https://pub.dev/packages/rhttp) by provide a "pure dart" implementation

**Comment by Tienisto** [comment]
Yes, it seems to be possible according to https://cjycode.com/flutter_rust_bridge/guides/miscellaneous/pure-dart

I am wondering if it still works on Flutter because https://github.com/flutter/flutter/issues/129757 is not resolved and https://github.com/dart-lang/sdk/issues/50565 is only available via experimental flag.

If we are lucky, then adding a `build.yaml` file (https://github.com/fzyzcjy/flutter_rust_bridge/blob/master/frb_example/pure_dart/build.dart) is enough to enable experimental Dart support.

**Comment by Tienisto** [comment]
I tried to add `build.dart` into the project but `flutter_rust_bridge_utils` is not published on pub.dev yet.

Ref: https://github.com/fzyzcjy/flutter_rust_bridge/issues/2274
